client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -48,10 +43,67 @@ public ReaderGroupConfigBuilder startingTime(long time) {
         *
         * @return Reader group config builder.
         */
-
        public ReaderGroupConfigBuilder disableAutomaticCheckpoints() {
            this.automaticCheckpointIntervalMillis = -1;
            return this;
        }
+
+       /**
+        * Add a stream and its associated start {@link StreamCut} to be read by the readers of a ReaderGroup.
+        * @param streamName Stream name.
+        * @param startStreamCut Start {@link StreamCut}
+        * @return Reader group config builder.
+        */
+       public ReaderGroupConfigBuilder stream(final String streamName, final StreamCut startStreamCut) {
```

Comment:
Will we add support for `endingStreamCuts` at a later time?   Would this be done as another overload of `stream`?

Reply:
yes, in this PR will add support only for `startingStreamCuts`.